do.sh

```diff
@@ -186,8 +186,8 @@ ovn_repo="${OVN_REPO:-https://github.com/ovn-org/ovn.git}"
 ovn_branch="${OVN_BRANCH:-main}"
 
 # ovn-fake-multinode env vars
-ovn_fmn_repo="${OVN_FAKE_MULTINODE_REPO:-https://github.com/ovn-org/ovn-fake-multinode.git}"
-ovn_fmn_branch="${OVN_FAKE_MULTINODE_BRANCH:-main}"
+ovn_fmn_repo="${OVN_FAKE_MULTINODE_REPO:-https://github.com/LorenzoBianconi/ovn-fake-multinode.git}"
+ovn_fmn_branch="${OVN_FAKE_MULTINODE_BRANCH:-ovn-ic-for-heater}"
```

Comment:
Can you please open a PR in the ovn-org/ovn-fake-multinode repo for this first?  I'd prefer if we keep ovn-heater pointing to the official main branch of ovn-fake-multinode.

Reply:
@dceara ack, this is the [PR](https://github.com/ovn-org/ovn-fake-multinode/pull/84) for ovn-fake-multinode